DigitalLearningSolutions.Web/ServiceFilter/VerifyAdminUserCanProceed.cs

```diff
@@ -21,7 +21,8 @@ public override void OnActionExecuting(ActionExecutingContext context)
                 return;
             }
 
-            if (context.HttpContext.User.Claims.FirstOrDefault(c => c.Type == "UserUserAdmin" && c.Value == "True") == null)
+            if (context.HttpContext.User.Claims.FirstOrDefault(c => c.Type == "UserUserAdmin" && c.Value == "True") == null
+                && context.HttpContext.User.Claims.FirstOrDefault(c => c.Type == "UserCentreAdmin" && c.Value == "True") == null)

```

Comment:
What is this code doing, @ColinBeebyHEE? (It's early and I haven't had enough coffee). Why are we looking at specific admin roles in the claim? There are more admin roles than these two and I'm guessing that we should be using OR rather than AND between them? But I'm probably missing something...

Reply:
@kevwhitt-hee OK code has been modified to check the UserAdminID rather than specific user admin roles.